### PR TITLE
feat: Added support for django3.2

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-2620

### CODE REVIEW

**Comment by Jawayria** [comment]
This warning is raised during the execution of tests and was not fixed under the scope of this PR
`RemovedInDjango40Warning: Passing None for the middleware get_response argument is deprecated.
`

**Review by mraarif — APPROVED** [review]

File: tox.ini [hunk 1]
```diff
@@ -15,6 +16,5 @@ commands = pytest --cov-branch  --cov-report=html:build/coverage/html --cov-repo
 commands =
 	pycodestyle --config=.pep8 lang_pref_middleware
 	pylint --rcfile=.pylintrc lang_pref_middleware
-	# Ignore module level docstrings and all test files
```
Comment (awais786) [inline tox.ini:18]: Revert this change.
